src/create.cpp

```diff
@@ -603,15 +613,39 @@ namespace create {
     }
   }
 
-  bool Create::isCliff() const {
```

Comment:
I think keeping this function is still useful. For instance, if the user is only interested in detecting a cliff for the purposes of stopping the robot (and does not care if it is left or right). But now the implementation can change:

```c
return (isCliffLeft() || isCliffFrontLeft() || isCliffFrontRight() || isCliffRight());
```